c2corg_ui/static/js/editing/outingediting.js

```diff
@@ -122,7 +124,10 @@ app.OutingEditingController.prototype.successRead = function(response) {
  */
 app.OutingEditingController.prototype.prepareData = function(data) {
   // adapt the Object for what's expected on the API side + format the outing
+  this.submit = true;
```

Comment:
Thanks for asking :)

That's because ``prepareData`` is now used by the preview request as well: https://github.com/c2corg/v6_ui/blob/3a080ad3536c30eb3b2f166e0d51b9d10e833b5d/c2corg_ui/static/js/editing/documentediting.js#L570-L574 (data sent to the preview service or to the saving/creating services should be consistent since the same template helpers are used to display them).

But ``this.submit`` is set to ``true`` only when submitting the form: https://github.com/c2corg/v6_ui/blob/3a080ad3536c30eb3b2f166e0d51b9d10e833b5d/c2corg_ui/static/js/editing/documentediting.js#L318

and we need it in the outing's ``formatOuting_`` method: https://github.com/c2corg/v6_ui/blob/3a080ad3536c30eb3b2f166e0d51b9d10e833b5d/c2corg_ui/static/js/editing/outingediting.js#L140-L141

I admit it would make sense to move ``this.submit = true;`` directly to the preview service caller in the parent class: https://github.com/c2corg/v6_ui/blob/3a080ad3536c30eb3b2f166e0d51b9d10e833b5d/c2corg_ui/static/js/editing/documentediting.js#L562
=> I think I will make the change.